### PR TITLE
Fix inline styles in Web/API - x

### PR DESCRIPTION
**Pull request by hamishwillee**
Part of fixing #5438

Window and XML cases.

### CODE REVIEW

File: files/en-us/web/api/xrinputsource/gripspace/index.html [hunk 1]
```diff
@@ -41,34 +41,17 @@ <h3 id="Value">Value</h3>
...
-      <div style="background-color: #FFF;"><img
-          alt="A diagram showing how the grip space indicates the local coordinate system for the player's hand relative to the world."
-          src="gripspace-righthand-light.svg"></div>
-    </figure>
-  </div>
-</div>
-
-<p class="cleared">As shown in the diagram above, the coordinate system is oriented as
-  follows:</p>
+<p><strong>The coordinate system for the left hand's grip space.</strong></p>
```
Comment (hamishwillee) [inline files/en-us/web/api/xrinputsource/gripspace/index.html:44]: FYI, this just renders figure captions as bold heading followed by image

File: files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html [hunk 2]
```diff
@@ -792,15 +792,10 @@ <h2 id="See_also">See also</h2>
...
+  <li><a href="/en-US/docs/Mozilla/Add-ons/Code_snippets/Timers">JavaScript timers</a></li>
   <li>{{domxref("WindowOrWorkerGlobalScope.setTimeout")}}</li>
   <li>{{domxref("WindowOrWorkerGlobalScope.clearTimeout")}}</li>
   <li>{{domxref("WindowOrWorkerGlobalScope.clearInterval")}}</li>
   <li>{{domxref("window.requestAnimationFrame")}}</li>
-  <li><a href="/en-US/Add-ons/Code_snippets/Timers/Daemons"><em>Daemons</em>
-      management</a></li>
+  <li><a href="/en-US/docs/Mozilla/Add-ons/Code_snippets/Timers/Daemons">Daemons management</a></li>
 </ul>
-
```
Comment (hamishwillee) [inline files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html:803]: FYI, this "gtx-trans" icon did not appear to render(did various tests) so I have just deleted it. Other changes above are just flaw fixes.

File: files/en-us/web/api/window/screenleft/index.html [hunk 3]
```diff
@@ -77,22 +77,8 @@ <h2 id="Examples">Examples</h2>
...
-  </thead>
-  <tbody>
-    <tr>
-      <td>{{ SpecName('CSSOM View', '#dom-window-screenx', 'Window.screenLeft') }}</td>
-      <td>{{ Spec2('CSSOM View') }}</td>
-      <td>Initial definition.</td>
-    </tr>
-  </tbody>
-</table>
+<p>{{Specifications}}</p>
```
Comment (hamishwillee) [inline files/en-us/web/api/window/screenleft/index.html:80]: Note, fixed a few cases of style in spec table just by adding this macro (I tested it works).

File: files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html [hunk 4]
```diff
@@ -792,15 +792,10 @@ <h2 id="See_also">See also</h2>
 
 <ul>
   <li>A polyfill of <code>setInterval</code> which allows passing arguments to the callback is available in <a href="https://github.com/zloirock/core-js#settimeout-and-setinterval"><code>core-js</code></a></li>
-  <li><a href="/en-US/Add-ons/Code_snippets/Timers">JavaScript timers</a></li>
+  <li><a href="/en-US/docs/Mozilla/Add-ons/Code_snippets/Timers">JavaScript timers</a></li>
```
Comment (wbamberg) [inline files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html:795]: This and the other Add-ons/Code_snippets one should be deleted.

**Review by wbamberg — DISMISSED** [review]
Looks great but I suggest removing a couple of old broken add-ons links. Thanks @hamishwillee !

File: files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html (hunk 4 above)
Comment (hamishwillee) [reply files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html:795]: Completely happy with that. There are lots of things I might do but I decided not to in order to make your review easier :-)

Comment (wbamberg) [reply files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html:795]: Yeah, I would not have commented except you touched them :). That whole page makes me cringe though.

Comment (hamishwillee) [reply files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html:795]: Done, ready to re-review :-)

Comment (hamishwillee) [reply files/en-us/web/api/windoworworkerglobalscope/setinterval/index.html:795]: Fair enough. Quite a bit of effort to fix up, and the queue keeps growing :-)

**Comment by wbamberg** [comment]
I didn't see the links removed, are you sure this was done?
https://github.com/mdn/content/pull/5517/files#diff-457e6ef5db81b8c470346dec79428f943d82d1af557b3ef3d2050eea747d19f6R487

**Comment by hamishwillee** [comment]
Apologies. Made change but must have forgotten to push it. Done now.

**Review by wbamberg — APPROVED** [review]

**Comment by wbamberg** [comment]
👍 thank you @hamishwillee !

**Pull request merged**